packages/next-preact/alias.js

```diff
@@ -1,7 +1,7 @@
-const path = require('path')
 const moduleAlias = require('module-alias')
 
 module.exports = () => {
-  moduleAlias.addAlias('react', path.join(__dirname, './preact-compat.js'))
-  moduleAlias.addAlias('react-dom', 'preact-compat')
+  moduleAlias.addAlias('react', 'preact/compat')
+  moduleAlias.addAlias('react-dom', 'preact/compat')
+  moduleAlias.addAlias('react-ssr-prepass', 'preact-ssr-prepass')
```

Comment:
As per my other comment, I'd prefer `preact-ssr-prepass` be a dependency of this plugin rather than being left up to userland.

```suggestion
  moduleAlias.addAlias('react-ssr-prepass', require.resolve('preact-ssr-prepass'))
```